react-forms/src/DatePicker.tsx

```diff
@@ -195,7 +207,7 @@ export default function DateTimePicker<T>(props: IProps<T>) {
                 step={step}
             />
             <div className="invalid-feedback">
-                {props.Feedback == null ? props.Field.toString() + ' is a required field.' : props.Feedback}
+                {feedbackMessage || props.Feedback || `${props.Field.toString()} is a required field.`}
```

Comment:
`feedbackMessage` is a `string` so this does not work. `props.Feedback` is also ` string` while technically `null` in a string is considered `false` this is not great from a readability perspective. `feedBackMessage` is never `null` so that will always take precedence in this case which means none of the other stuff matters. That's the kind of error you will overlook by using this shortcut. I recommend making these very clear `if` statements (or `condition ? x : y`) which would make those issues obvious